api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java

```diff
@@ -152,6 +170,21 @@ public CohortDefinition getJ() {
     return cd;
   }
 
+  /** Fetches patients in the first line of treatment */
+  private CohortDefinition getPatientsInTheFirstLineOfTreatment() {
```

Comment:
@Edgencio Can you please name this function differently, it's not only getting the first line of treatment...
 ```suggestion
  private CohortDefinition getPatientsInTheFirstOrWithoutLineOfTreatment() {
```